### DISCUSSION TITLE
Не рабоает exec в форматировании

### DISCUSSION THREAD

**Discussion by nshy** [totumonline/totum-ru-issues-and-discussions#570] [ОШБК!] [answered]
Например:

    p1=: panelHtml(html: $html)
    html: exec(code: $code)
    ```code:totum
    =: 'foo'
    ```

Не добавляет ничего к контекстной панели.

`totum-mit 4.11.57.0`

Хотелось бы `code` в примере выше переиспользовать в разных таблицах, но сейчас это невозможно.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Да, понятно почему не работает. Exec был сделан задолго до появления секций `p` и поэтому исполняется в f-режиме. Точнее сейчас он исполняется в режиме того кода из которого вызывается. Для форматирования поля это `f`.

Я записал, но в ближайшее время мы не сделаем это.
